bundle/src/main/java/com/adobe/acs/commons/dam/impl/ReviewTaskAssetMoverHandler.java

```diff
@@ -229,13 +360,45 @@ private void moveAsset(AssetManager assetManager, Asset asset, ValueMap taskProp
 
             if (destPath != null) {
                 if (StringUtils.startsWith(destPath, PATH_CONTENT_DAM)) {
-                    assetManager.moveAsset(asset.getPath(), destPath + "/" + asset.getName());
-                    log.info("Moved [ {} ] ~> [ {} ] based on approval status [ " + status + " ]",
-                            asset.getPath(), destPath + "/" + asset.getName());
+
+                    String destAssetPath = destPath + "/" + asset.getName();
+                    final boolean exists = assetManager.assetExists(destAssetPath);
+
+                    if (exists) {
+                        if (StringUtils.equals(asset.getPath(), destAssetPath)) {
+                            log.info("Reviewed asset [ {} ] is already in its final location, so there is nothing to do.", asset.getPath());
+                        } else if(CONFLICT_RESOLUTION_REPLACE.equals(conflictResolution)) {
+                            assetManager.removeAsset(destAssetPath);
+                            resourceResolver.commit();
+                            assetManager.moveAsset(asset.getPath(), destAssetPath);
+                            log.info("Moved with replace [ {} ] ~> [ {} ] based on approval status [ " + status + " ]",
```

Comment:
There is a String, Object[] which is resolved when more than 2 placeholder args are used. http://www.slf4j.org/apidocs/org/slf4j/Logger.html#info(java.lang.String, java.lang.Object...)

Different type of overhead: String concat vs Object[] creation. The ... API is cleaner IMO